lib/iris/tests/unit/coords/test_Cell.py

```diff
@@ -120,22 +143,87 @@ def test_PartialDateTime_other(self):
 
 
 class Test_contains_point(tests.IrisTest):
```

Comment:
OK in the agreed absence of parametrization, I have made my own matrix and I think a few test cases are missing?

| `Cell` ⬇ &nbsp;&nbsp;&nbsp;&nbsp; Point ➡  | `datetime` | `PartialDateTime` | `cftime.datetime` | `cftime.datetime(calendar="360_day")`  |
|---|---|---|---|---|
| `datetime` |   | ✔ | ✔ | ✔ |
| `cftime.datetime` | ✔ | ✔ |   |   |
| `cftime.datetime(calendar="360_day")` | ✔ | ✔ |   |   |

Is this accurate?

Reply:
They are, but I didn't write cases for matching cases (datetime/datetime, cftime/cftime, 360day/360 day) on purpose, because the intent of the test case is to check operations between different object classes, not within the same class (on the assumption that is tested by those libaries/classes themselves). Same as cftime.datetime/cftime.datetime(calendar="360_day"), assuming that within a object type calendar operations can be assumed and we just wanted to check across object types. 

I've put in a test for the 360day/360day case, because while it should be fine it is a check that iris handles non-standard calendars if they do match, which is probably worthwhile to include.